app/views/content/what-is-a-pgce.md

```diff
@@ -51,6 +51,8 @@ Getting QTS only means that you’re qualified to teach in most schools in Engla
 
```

Comment:
I think line 50 (in prod already!) is misleading in that it says `Getting QTS **only**`

Should we change to `Getting QTS only or QTS with PGCE......`? @EllieNodder